.npmrc

```diff
@@ -1,14 +0,0 @@
-# Enforce Node and pnpm versions as specified in `package.json`
-engine-strict=true
-
-# Make installation of peer dependencies more predictible
-auto-install-peers=false
-strict-peer-dependencies=true
-
-# Save exact dependency versions in `package.json`
-save-prefix=""
-
-# Branch that must be checked out when publishing
-publish-branch=main
-
-scope=@h5web
```

Comment:
Turns out [`scope`](https://docs.npmjs.com/cli/v9/using-npm/config#scope) is not really relevant for us. The `publish` command just takes the package name from `package.json`.